mlrun/serving/v2_serving.py

```diff
@@ -542,48 +542,67 @@ def _init_endpoint_record(
         function_uri=graph_server.function_uri, versioned_model=versioned_model_name
     ).uid
 
+    model_ep = None
     # If model endpoint object was found in DB, skip the creation process.
     try:
-        mlrun.get_run_db().get_model_endpoint(project=project, endpoint_id=uid)
-
+        model_ep = mlrun.get_run_db().get_model_endpoint(
+            project=project, endpoint_id=uid
+        )
     except mlrun.errors.MLRunNotFoundError:
         logger.info("Creating a new model endpoint record", endpoint_id=uid)
```

Comment:
this log will appear even if tracking is not enabled, which is wrong in this case.
also, I think having the mep creation logic inside exception handling is confusing - better to raise a flag and then do the mep creation based on the flag value.